docs/guide/search/aggregation_reference/locationchildrenterm_aggregation.md

```diff
@@ -0,0 +1,16 @@
+# LocationChildrenTermAggregation
+
+The [LocationChildrenTermAggregation](https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Publish/API/Repository/Values/Content/Query/Aggregation/Location/LocationChildrenTermAggregation.php) aggregates search results by the number of children of a Location.
+
+## Arguments
+
+- `name` - name of the Aggregation object
+
+## Example
+
+``` php
+$query = new Query();
```

Comment:
I think it should be:
```suggestion
$query = new LocationQuery();
```
As this is a Location Aggregation (only supported for Location Queries)